bin/oref0-autotune.sh

```diff
@@ -217,8 +217,8 @@ do
     
     # Autotune  (required args, <autotune/glucose.json> <autotune/autotune.json> <settings/profile.json>), 
     # output autotuned profile or what will be used as <autotune/autotune.json> in the next iteration
-    echo "oref0-autotune-core autotune.$i.json profile.json profile.pump.json > newprofile.$i.json"
-    if ! oref0-autotune-core autotune.$i.json profile.json profile.pump.json > newprofile.$i.json; then
+    echo "oref0-autotune-core autotune.$i.json profile.json profile.pump.json | jq . -M > newprofile.$i.json"
+    if ! oref0-autotune-core autotune.$i.json profile.json profile.pump.json | jq . -M > newprofile.$i.json; then
```

Comment:
I believe -M is unnecessary here as well, as jq only outputs colored json by default if writing to a terminal.